addons/web/static/src/views/pivot/pivot_view.scss

```diff
@@ -21,79 +21,42 @@
             padding: 0!important;
         }
         background-color: $o-view-background-color;
-        width: auto;  // bootstrap override
 
         // Inform the user that he may click on a cell to be redirected to a list view of the
         // items corresponding to the clicked cell
         &.o_enable_linking {
             .o_pivot_cell_value:not(.o_empty):hover {
                 color: $o-brand-primary;
                 cursor: pointer;
```

Comment:
Across the UI we have tons of elements that need the  `cursor: pointer;` rule...
Could you create an utility class in `ui.scss` and apply it here ?

For the color-on-hover variation you can use the mixin https://github.com/odoo/odoo/blob/master/addons/web/static/src/legacy/scss/utils.scss#L182